docs/user/algo_ppo.md

```diff
@@ -0,0 +1,44 @@
+| **Framework(s)**  | .. figure:: ./images/pytorch.png                                                                               | .. figure:: ./images/tensorflow.png                                                                      |
+|                   |    :scale: 10%                                                                                                 |    :scale: 20%                                                                                           |
+|                   |                                                                                                                |                                                                                                          |
+|                   |    PyTorch                                                                                                     |    TensorFlow                                                                                            |
++-------------------+----------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------+
+| **API Reference** | `garage.torch.algos.PPO <../_autoapi/garage/torch/algos/index.html#garage.torch.algos.PPO>`_                   | `garage.tf.algos.PPO <../_autoapi/garage/tf/algos/index.html#garage.tf.algos.PPO>`_                      |
++-------------------+----------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------+
+| **Code**          | `garage/torch/algos/ppo.py <https://github.com/rlworkgroup/garage/blob/master/src/garage/torch/algos/ppo.py>`_ | `garage/tf/algos/ppo.py <https://github.com/rlworkgroup/garage/blob/master/src/garage/tf/algos/ppo.py>`_ |
++-------------------+----------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------+
+| **Examples**      | `examples <algo_ppo.html#examples>`_                                                                                                                                                                                      |
++-------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+Proximal Policy Optimization Algorithms (PPO) is a family of policy gradient
+methods which alternate between sampling data through interaction with the
+environment, and optimizing a "surrogate" objective function using stochastic
+gradient ascent.
+
+## Examples
```

Comment:
Please include a few more examples (at least a couple each for torch and TF).

You should mention the different entropy regularization and maximum entropy implemented with PPO.